src/test/java/com/ensao/gi5/lint/runner/RunnerTest.java

```diff
@@ -3,12 +3,14 @@
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
+import java.io.IOException;
+
 public class RunnerTest {
 
     public static final String ARG_PARAM = "-s";
 
     @Test
-    public void testMain() {
+    public void testMain() throws IOException {
```

Comment:
This tests hangs, because it waits for input from the stdin ( standard input).
try something like in this example 
https://github.com/BelmoMusta/battle-dev/blob/master/src/test/java/musta/belmo/battledev/AbstractTestClass.java